packages/kernels/mocaccino-lts/build.yaml

```diff
@@ -4,6 +4,8 @@ env:
 - GCC_VERSION=10
 - ARCH={{.Values.arch}}
 - KERNEL_PREFIX=vanilla
+- SUFFIX=mocaccino
```

Comment:
Is there a reason why you moved hardcoded vars away from `definition.yaml`?

Reply:
Because:
1. I set the env there that I use directly in the other packages without set, again and again, the same option.
2. They are mocaccino kernel specs, no needed to have this field modifiable.

Reply:
1) Don't rely on that please unless unnecessary. All the env should be visible in the specs, it adds complexity and it's not obvious for unexperienced readers.
2) You just move the hardcoded from the `definition.yaml` inside `build.yaml`, with no real catch. `definition.yaml` should contain metadata that can be freely edited, so I would expect them there